### ISSUE TITLE
No need for completion animation when not logged in

### ISSUE THREAD

**Issue by RandellDawson** [freeCodeCamp/freeCodeCamp#37878]
If a user is not logged in, the newly implemented completion animation along with the section name above it (see below) are not needed.  Also, it will always show `0% complete` which does not really serve any purpose for the end user.  I suggest hiding this feature unless logged in.

![image](https://user-images.githubusercontent.com/5313213/70304939-e6fcc280-17b7-11ea-9213-4fe93497838d.png)

**Comment by thecodingaviator** [comment]
Where does the code to this modal component lie?

**Comment by moT01** [comment]
although, it does say "sign in to save your progress" just below - so maybe there's a little more incentive to create an account.

**Comment by ojeytonwilliams** [comment]
It would need to have a non-zero percentage to make sense.  As it stands, it says "you've made no progress, sign in to save it".  

Come to think of it, this would be a nice candidate for A/B testing.  In practice, what does result in more people signing in? Just the button or the button with a hint that you could be making progress?

**Comment by bigkatspence** [comment]
to me what makes sense is a button with a hint that tells the student to
login to save and if not registered a smaller tab saying to the student
register here !

**Comment by bigkatspence** [comment]
either way ,a win-win

**Comment by moT01** [comment]
So what about all these...
![Screen Shot 2019-12-13 at 9 21 47 PM](https://user-images.githubusercontent.com/20648924/70842851-a4a63780-1dee-11ea-9459-56ffe7c660e1.png)
They will also always to show `0` for unauthed users.

**Comment by thecodingaviator** [comment]
@moT01 I think the best way is to open a new issue for that and I’ll take that up too.

**Comment by moT01** [comment]
I dunno, I'm still not convinced that either of these need to be removed for non-authed users. I really think it will give them more incentive to create an account.

**Comment by bigkatspence** [comment]
Same here!

Kamp Busker/Josh/Sdit Jax

**Comment by raisedadead** [comment]
We discussed about this a bit internally and I think we would want to leave the progress here and everywhere else as is. We checked that the count of users who did not sign in (wanting to continue anonymously) is insignificant from those who signed in to save progress.

Instead, I think showing the progress at `0%` here and everywhere else drives the intuition that something is wrong and triggers the intent to check and sign in. This also goes for the challenge map timeline etc.

User who want to remain anon are rare and also mostly aware of the fact that they are not signed in.

I think we should close this as is, except I would want to change the color of the sign in CTA to the orange color for making it standout.
